sermant-backend/src/main/java/io/sermant/backend/common/conf/CommonConst.java

```diff
@@ -139,14 +139,14 @@ public class CommonConst {
                     + "---  " + System.lineSeparator();
 
     /**
-     * Service name for global configuration
+     * Plugin type for non-Semant plugins
      */
-    public static final String GLOBAL_CONFIGURATION_SERVICE_NAME = "ALL_SERVICE";
+    public static final String OTHER_PLUGIN = "other";
 
     /**
-     * Default regular expression for configuration item Group
+     * Wildcard
      */
-    public static final String CONFIGURATION_DEFAULT_PATTERN = "^(app=[^&]*+&environment=[^&]*+(&service=[^&]*)?)?$";
+    public static final String WILDCARD = "[^&]+";
```

Comment:
WILDCARD should  be *

Reply:
Prevents special characters. Therefore, * is not used.